components/dashtable.js

```diff
@@ -60,9 +88,31 @@ export default function DashTable(props) {
             course.allChallenges.length
           }`;
         }
-      } else {
+      } else if (courseSelector == 'student-name') {
         studentCompletionData[courseSelector] = studentName;
+      } else if (courseSelector == 'student-activity') {
+        // studentCompletionData[courseSelector] = `${studentActivityData}`;
+        if (studentActivityData >= 2) {
```

Comment:
you can refactor lines 95 -113 into a separate react component to make it more testable

Reply:
We've moved this part into a separate react component!